propolis/src/hw/bhyve/atpit.rs

```diff
@@ -0,0 +1,35 @@
+use std::sync::Arc;
+
+use crate::dispatch::DispCtx;
+use crate::inventory::Entity;
+use crate::migrate::Migrate;
+
+use erased_serde::Serialize;
+
+pub struct BhyveAtPit {}
```

Comment:
Similarly here, I'd expect providing tsc or at the very least hpet rather than pit.

Reply:
Again, it's emulated in kernel, so we'll need to migrate the state.  The PIT is often used for initial timer calibration at startup.